agent/rpm/pbench-agent.rpmlintrc

```diff
@@ -0,0 +1 @@
+../../utils/rpmlint
```

Comment:
Are these really necessary if `rpmlint` is looking for the config file at `${PBENCHTOP}/utils` already?? Or is there something weird(er)(er) about this `XDG_CONFIG_HOME`???

Reply:
The new version of `rpmlint` looks for `${XDG_CONFIG_HOME}/rpmlint/*toml`, which it won't find because we have `${XDG_CONFIG_HOME}/rpmlint` as the configuration file for the previous version.

So this symlinks are auto-discovered by `rpmlint` in the directory in which it is invoked.

Reply:
In the [old version](https://linux.die.net/man/1/rpmlint), it looks for the config file at `$XDG_CONFIG_HOME/rpmlint`.

In the [new version](https://github.com/rpm-software-management/rpmlint#configuration) the config file can come from a number of places, one being any `*.rpmlintrc` or `*-rpmlintrc` located in the same directory as the specfile. (The best practice apparently is to use the name `$PACKAGE_NAME.rpmlintrc`.)  (This file uses syntax compatible with the old version.)

Given that the old version doesn't read TOML format, I think Peter's approach is probably as good as it gets.